feat: Information highlight (#2961)
Workday/canvas-kit: modules/preview-react/information-highlight/README.md

> 
> # Canvas Kit Information Highlight
> 
> A layout that will seperate out text or other information  to make it more prominent to the user
> 
> View the [documentation for Information Highlight](https://workday.github.io/canvas-kit/?path=/docs/information-highlight-react)
> on Storybook.
> 
> [> Workday Design Reference: Information Highlight](https://design.workday.com/components/)

Reviewer: ```suggestion
# Canvas Kit Preview Information Highlight

View the [documentation for Information Highlight](https://workday.github.io/canvas-kit/?path=/docs/preview-information-highlight--docs)
on Storybook.
```
